tests_e2e/scripts/execute_tests_container.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+az login --service-principal --username "$AZURE_CLIENT_ID" --password "$AZURE_CLIENT_SECRET" --tenant "$AZURE_TENANT_ID" > /dev/null
+
+az acr login --name waagenttests
+
+docker pull waagenttests.azurecr.io/waagenttests:latest
+
+# Logs will be placed in this location. Make waagent (UID 1000 in the container) the owner.
+mkdir "$HOME/logs"
+sudo chown 1000 "$HOME/logs"
+
+docker run --rm \
+      --volume "$BUILD_SOURCESDIRECTORY:/home/waagent/WALinuxAgent" \
+      --volume "$DOWNLOADSSHKEY_SECUREFILEPATH:/home/waagent/id_rsa" \
```

Comment:
$BUILD_SOURCESDIRECTORY and $DOWNLOADSSHKEY_SECUREFILEPATH are inputs to the container, while $HOME/logs is its output.

We should not modify the input directories, since the may be coming from a developer's machine while debugging/etc (the script that executes the tests creates a file under '/home/waagent' and changes the mode of '/home/waagent/id_rsa')

Reply:
I see your point. If the file semantics changes in container path would change the input path too. Makes sense.